### ISSUE TITLE
/keybase: No such file or directory

### ISSUE THREAD

**Issue by fladd** [keybase/kbfs#1368]
I installed keybase on Mac OS from the installer on your website. I then added the mac as a new device. According to `keybase status` KBFS is running (version 1.0.2-20171122003301+247bbfc6).

However, it is not accessible from the terminal, as your documentation suggests it should be:

```
cd /keybase
-bash: cd: /keybase: No such file or directory

cd /keybase/private
-bash: cd: /keybase/private: No such file or directory

cd /keybase/public
-bash: cd: /keybase/public: No such file or directory
```

**Comment by strib** [comment]
@fladd: check the Folders tab of the Keybase GUI.  There might be a link for installing the kernel extension, which is required for file system access now.  Recent macOS changes forced us to not install the kext by default, so now you have to do it afterwards.

**Comment by fladd** [comment]
@strib Indeed, that was the case. Might be a good idea to add this to the installation instructions.
